editor/editor_settings.cpp

```diff
@@ -68,6 +69,9 @@ bool EditorSettings::_set(const StringName &p_name, const Variant &p_value) {
 	if (changed && initialized) {
 		changed_settings.insert(p_name);
 		emit_signal(SNAME("settings_changed"));
+		if (p_name == "text_editor/external/exec_path") {
+			_exec_path_changed(p_value);
```

Comment:
This changes the path again, so it should happen before emitting the signal.

Reply:
It is not changing the path, it should change the `exec_flags`.
Where would you suggest to place it?
<img width="726" alt="image" src="https://github.com/user-attachments/assets/23b2079f-bf0c-4323-a71e-d6d55d57237e" />

Reply:
Ah right. But it's still modifying a setting, so set it before the signal.

Reply:
done, thank you for the pointer.